configs/SC011_0219190_CatGT-input.json

```diff
@@ -1,8 +1,8 @@
 {
     "catGT_helper_params": {
         "car_mode": "gblcar",
-        "catGTPath": "C:\\Users\\labadmin\\Documents\\jic\\CatGT-win",
-        "cmdStr": "-prb_fld -out_prb_fld -aphipass=300 -gfix=0.4,0.10,0.02 -tshift-SY=0,-1,6,500-XA=0,1,3,500 -iXA=1,3,3,0  -XD=-1,1,50 -XD=-1,2,1.7 -XD=-1,3,5 -iXD=-1,3,5",
+        "catGTPath": "/home/ubuntu/neuropixel/CatGT-linux",
```

Comment:
Will there some sort of path handling or root path concatenation at some point later?

Reply:
This depends on the MATLAB toolbox implementation. Currently, there is an available option to load from .env file/env variable